ios/Pods/Local Podspecs/shared.podspec.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
We can gitignore `/Pods`, it will rebuild every time. Or you can keep it to speed up CICD (without including `pod install` in CICD), but I'm not sure how much the speed up is with KMM module as well (KMM runs rebuild of pod every run).

Reply:
@blyscuit I rebased to ignore `/Pods` folder in this commit https://github.com/luongvo/kmm-survey/pull/39/commits/9a9a6d8dfdff542a4261734c73e3194b5bb55c75.